src/test/kotlin/org/neo4j/graphql/utils/CypherTestSuite.kt

```diff
@@ -36,7 +36,7 @@ class CypherTestSuite(fileName: String) : AsciiDocTestSuite(fileName) {
         val result = createTransformationTask(title, schema, codeBlocks)
 
         val tests = mutableListOf<DynamicNode>()
-        if (DEBUG) {
+        if (System.getProperty("debug", "false") == "true") {
```

Comment:
Probably more specific name? Like `neo4j-graphql-java.debug` Otherwise it might conflict with others?

Reply:
I also meant to set it in the companion object so that it doesn't have to read it time and again, but it's probably not a biggie.